tests/Suggestor/Suggestor.test.ts

```diff
@@ -2,16 +2,31 @@
  * @jest-environment jsdom
  */
 import moment from 'moment';
+import * as chrono from 'chrono-node';
 import { getSettings } from '../../src/Config/Settings';
 import type { SuggestInfo } from '../../src/Suggestor';
 import { makeDefaultSuggestionBuilder } from '../../src/Suggestor/Suggestor';
 import { DEFAULT_SYMBOLS } from '../../src/TaskSerializer/DefaultTaskSerializer';
 
 window.moment = moment;
 
+// Set predictable date for all tests in this file
+const mockDate = new Date(moment('2022-07-11 15:00').valueOf());
+
+jest.spyOn(chrono, 'parseDate').mockImplementation(
+    (text, _, options) => chrono.en.casual.parseDate(text, mockDate, options)!,
```

Comment:
`chrono.parseDate` is a shortcut to `chrono.en.casual.parseDate` so the result is the same.